backend/src/resolvers/posts.js

```diff
@@ -1,6 +1,43 @@
-import { neo4jgraphql } from 'neo4j-graphql-js'
+import { neo4jgraphql, cypherQuery } from 'neo4j-graphql-js'
+import { v1 as neo4j } from 'neo4j-driver'
+import _ from 'lodash'
+
+export function extractQueryResult ({ records }, returnType) {
+  const variableName = 'post'
+  let result = null
+  result = records.map(record => record.get(variableName))
+  // handle Integer fields
+  result = _.cloneDeepWith(result, field => {
+    if (neo4j.isInt(field)) {
+      // See: https://neo4j.com/docs/api/javascript-driver/current/class/src/v1/integer.js~Integer.html
+      return field.inSafeRange() ? field.toNumber() : field.toString()
+    }
+  })
+  return result
+}
 
 export default {
+  Query: {
+    Post: async (object, params, context, resolveInfo) => {
+      let [query, cypherParams] = cypherQuery(params, context, resolveInfo)
+
+      // HACK-ATTACK
+      query = query.replace('RETURN', 'WITH `post` ORDER BY `post`.createdAt DESC RETURN')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T23:24:38Z" title="Friday, May 3rd 2019, 1:24:38 am +02:00">May 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,43 @@
-import { neo4jgraphql } from 'neo4j-graphql-js'
+import { neo4jgraphql, cypherQuery } from 'neo4j-graphql-js'
+import { v1 as neo4j } from 'neo4j-driver'
+import _ from 'lodash'
+
+export function extractQueryResult ({ records }, returnType) {
+  const variableName = 'post'
+  let result = null
+  result = records.map(record => record.get(variableName))
+  // handle Integer fields
+  result = _.cloneDeepWith(result, field => {
+    if (neo4j.isInt(field)) {
+      // See: https://neo4j.com/docs/api/javascript-driver/current/class/src/v1/integer.js~Integer.html
+      return field.inSafeRange() ? field.toNumber() : field.toString()
+    }
+  })
+  return result
+}
 
 export default {
+  Query: {
+    Post: async (object, params, context, resolveInfo) => {
+      let [query, cypherParams] = cypherQuery(params, context, resolveInfo)
+
+      // HACK-ATTACK
+      query = query.replace('RETURN', 'WITH `post` ORDER BY `post`.createdAt DESC RETURN')
```


Here's the hack :neckbeard:

Reply:
> [<img alt="appinteractive" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/appinteractive) **Authored by [appinteractive](https://github.com/appinteractive)**
_<time datetime="2019-05-07T08:48:57Z" title="Tuesday, May 7th 2019, 10:48:57 am +02:00">May 7, 2019</time>_

---


Let’s add a `// TODO: unhack hack-attack` 🦹🏼‍♂️🤣